src/lt/macros.clj

```diff
@@ -47,30 +47,30 @@
     [(first body) (rest body)]
     [[] body]))
 
-(defmacro on [name & body]
+(defmacro ^:private on [name & body]
```

Comment:
I am aware of :no-doc. My intent is to keep as much private as possible until we know we want them to be public